imports/plugins/core/ui/client/containers/buttonSelectContainer.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component, PropTypes } from "react";
+import classnames from "classnames";
+import { composeWithTracker, merge } from "/lib/api/compose";
+import Button from "../components/button/button.jsx";
+import { ButtonSelect } from "../components/button";
+
+class ButtonSelectContainer extends Component {
+  constructor(props) {
+    super(props);
+    const currentButton = <Button
```

Comment:
all logic around `currentButton` should be in the render function, not set on the state. I like to think of state like something serializable, so no JSX, no functions, only strings, objects, arrays, booleans, numbers ( basically like JSON).